src/hardware/TX/Jumper T-20 2400.json

```diff
@@ -0,0 +1,23 @@
+{
+    "serial_rx": 3,
+    "serial_tx": 1,
+    "radio_busy": 21,
+    "radio_dio1": 4,
+    "radio_dio2": 22,
```

Comment:
This does not appear to be connected on the schematic?

Reply:
checked.